src/modules/Resize/Module.js

```diff
@@ -20,13 +22,18 @@ module.exports = function Resize(options, UI) {
 
     function extraManipulation(pixels) {
       // Value above 100% scales up, and below 100% scales down
-      const resize_value = parseInt(options.resize.slice(0, -1));
+      const resize_height = Number(options.height);
+      const resize_width = Number(options.width);
+      const checkbox = Number(options.checkbox);
 
-      if (resize_value == 100) return pixels;
+      if (resize_height == 100) return pixels;
```

Comment:
This should be changed as we cant return the pixels if the new height is 100 because original image height may not be 100